storage-node-v2/src/services/webApi/controllers/publicApi.ts

```diff
@@ -138,10 +120,7 @@ export async function uploadFile(
 /**
  * A public endpoint: creates auth token for file uploads.
  */
-export async function authTokenForUploading(
-  req: express.Request,
-  res: express.Response
-): Promise<void> {
+export async function authTokenForUploading(req: express.Request, res: express.Response): Promise<void> {
```

Comment:
Also a `410` here.

One example of why I think it's not valid is that `getAccount` can potentially throw `No Joystream account loaded.`, which is not a request error, it's an error with node configuration. Same goes for `getApi`. There may also be other errors than will end up in `catch` block, so we're not really sure what was the cause of the `throw`.

`410` response means that _access to the target resource is no longer available at the origin server and that this condition is likely to be permanent._ ([source](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/410)). It may end up beeing cached (even forever!) by the client and the request will not be repeated again even after the issue is resolved,

Reply:
Introduced fine-grained error distinction.